### DISCUSSION TITLE
Shopify function not accepting new asset url formats

### DISCUSSION THREAD

**Discussion by andrewchen5678** [Shopify/function-examples#563] [Feedback: Cart Transform API]
Shopify cart transform function is not accepting the new asset url format that start with shop.example.com/cdn even if the old one is deprecated and the new one is returned by default on the frontend, which prevents images from being able to be used https://changelog.shopify.com/posts/changes-to-asset-urls

**Comment by moraleslevi** [comment]
This has been fixed. You can now use the correct CDN URLs on to the shop's domain.
